docs/UserGuide.md

```diff
@@ -169,15 +226,23 @@ It will not take effect immediately. A confirmation prompt will allow you to con
-| list a recipe        | `list`                                                                                                                                                                                                                 |
-| view specific recipe | `view <recipe number from list>`                                                                                                                                                                                       |
+
+### How to read this table?
+1. User actions (what you would want to do) is on the left while the command associated to it is on the right.
+2. Almost all the details of the commands are needed to be given before the command can be executed properly.
+   1. **Compulsory** details are denoted by angled-brackets such as \<mandatory details>.
+   2. **Optional** details of the command that are enclosed in square brackets such as [optional details].
+3. What does it mean when the input command says "... separated by commas"?
+
+
+| Action               | Command format                                                                                                                                                                                                 |
+|----------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| add a new recipe     | `add -n <name of recipe> -d <prep time of recipe> -ss <serving size/portions of recipe> -i <ingredients separated by commas> -s <steps separated by commas> [-t <tags separated by commas>]`                   |
+| edit a recipe        | `edit <name of recipe> -n <name of recipe> -d <prep time of recipe> -ss <serving size/portions of recipe> -i <ingredients separated by commas> -s <steps separated by commas> [-t <tags separated by commas>]` |
+| delete a recipe      | `delete -x <recipe number from list>` <br> or <br> `delete <recipe name>`                                                                                                                                      |
+| find a recipe        | `find <keyword> [other keywords separated by commas]`                                                                                                                                                          |
+| list a recipe        | `list`                                                                                                                                                                                                         |
+| view specific recipe | `view <recipe number from list>` <br/> or <br/> `view <recipe name>`                                                                                                                                           |
```

Comment:
Should change this to make it consistent with delete command
```suggestion
| view specific recipe | `view -x <recipe index from list>` <br/> or <br/> `view <recipe name>`                                                                                                                                           |
```

Reply:
went to edit the delete command instead since the command explanation follows the name-then-index slow